### DISCUSSION TITLE
Selective Target Transformations (Differencing Transforms)

### DISCUSSION THREAD

**Discussion by jrodenbergrheem** [JoaquinAmatRodrigo/skforecast#862] [Ideas]
Just an idea to have support to pass a dict to differentiation parameter similar to the way you can pass one to the transformer_series parameter, so when forecasting multiple series we can transform them selectively rather than having to difference all skus in a dataset!

**Comment by JavierEscobarOrtiz** [comment]
@jrodenbergrheem Thanks for the suggestion! Absolutely agree, we have noted for the next release 😄
